### PR TITLE
Add support for logarithmic quantities and the units neper and bel #132

### PR DESCRIPTION
**Pull request by l-johnston**

### CODE REVIEW

**Comment by ngoldbaum** [comment]
So I don't have much experience with these logarithmic units. Does it make sense for someone to do:

```
from unyt import dB

10**(300*dB)
```

Because right now that will raise an error, I think.

File: unyt/_unit_lookup_table.py [hunk 1]
```diff
@@ -426,6 +430,8 @@
         ("psi", ("pounds_per_square_inch",)),
         # dimensionless stuff
         ("dimensionless", ("_", "")),
+        ("B", ("bel",)),
```
Comment (ngoldbaum) [inline unyt/_unit_lookup_table.py]: Does `decibel` work as a unit name now or does that need special treatment?

File: unyt/tests/test_units.py [hunk 2]
```diff
@@ -802,3 +802,15 @@ def test_equal_has_same_hash():
...
     assert hash(a) == hash(b)
     assert hash(b) == hash(c)
+
+
+def test_bel_neper():
+    assert Unit("B").dimensions == Unit("Np").dimensions
+    a = 1 * Unit("B") / (np.log(10) / 2)
+    assert_allclose_units(a.to("Np"), 1 * Unit("Np"))
+    a = 2 * Unit("B")
+    b = 20 * Unit("decibel")
```
Comment (ngoldbaum) [inline unyt/tests/test_units.py]: Ah I see it does :)

**Comment by l-johnston** [comment]
As in all other cases, it is improper to raise a value to quantity and so the error is correct. I state in the usage doc to strip the unit using unyt_array.v before the power operation.

File: docs/usage.rst [hunk 3]
```diff
@@ -224,6 +224,29 @@ The one exception to this rule is for trigonometric functions applied to data wi
   >>> np.sin(45.*degree)
   array(0.70710678)
 
+Logarithmic Quantities and Units
+********************************
+
+The logarithmic quantities level-of-power and level-of-field and the units neper and
+bel are supported. In the next example, we represent the power measurements, p, as
+a logarithmic quantity at reference level, p_ref, in the units decibel.
```
Comment (ngoldbaum) [inline docs/usage.rst]: `p` and `p_ref` need double backticks, right now I think the underscore in p_ref will cause formatting issues

File: docs/usage.rst [hunk 4]
```diff
@@ -224,6 +224,29 @@ The one exception to this rule is for trigonometric functions applied to data wi
...
+  >>> dB.dimensions
+  (logarithmic)
+  >>> p = [1, 100]*mW
+  >>> p_ref = 1*mW
+  >>> level_of_power = 10*np.log10(p/p_ref)*dB
+  >>> level_of_power
+  unyt_array([ 0., 20.], 'dB')
+
+You can convert the logarithmic quantity back to physical units through exponentiation,
+just remember to remove the units using the unyt_array.v property.
```
Comment (ngoldbaum) [inline docs/usage.rst]: `unyt_array.v` needs either double backticks or, even better, to be a cross-reference to the API docs.

File: unyt/_physical_ratios.py [hunk 5]
```diff
@@ -160,3 +160,7 @@
 pascal_per_atm = 101325.0
 m_per_inch = 0.0254
 m_per_ft = 0.3048
+
+# logarithmic units
+# IEC 60027-3
```
Comment (ngoldbaum) [inline unyt/_physical_ratios.py]: Can you elaborate in this comment more what this is referring to? (e.g. with a link)

File: docs/usage.rst (hunk 3 above)
Comment (l-johnston) [reply docs/usage.rst]: done.

File: docs/usage.rst (hunk 4 above)
Comment (l-johnston) [reply docs/usage.rst]: done.

File: unyt/_physical_ratios.py (hunk 5 above)
Comment (l-johnston) [reply unyt/_physical_ratios.py]: done.

**Pull request merged**